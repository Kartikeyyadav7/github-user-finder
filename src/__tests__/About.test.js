import React from "react";
import { create } from "react-test-renderer";
import About from "../About";

test("snapshot", () => {
  const c = create(<About />);
  expect(c.toJSON()).toMatchSnapshot();
});
