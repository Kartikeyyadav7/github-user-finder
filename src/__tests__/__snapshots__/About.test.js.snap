// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="about"
>
  <h4>
    Hi, my name is Michal.
  </h4>
  <p>
    I'm an aspiring Web Developer, and I'm passionate about building intuitive and beautiful UI's.
  </p>
  <p>
    I'm an Ex-journalist and self taught programmer. I began my career transition a couple of years ago as part of a group called SheCodes. Since then, I've leanred HTML, CSS, JavaScript, and Python, which lead me to a role in a startup that teaches Data Science. I helped students make a similar transition into computer science related fields, which was extremely rewarding. I fell in love with working on websites' code and discovering their structure, and my goal is to keep learning so that I could build simple yet beautiful UI's.
  </p>
</div>
`;
